survey-runner.tf

```diff
@@ -236,6 +332,31 @@ module "survey-runner-database" {
   database_name                    = "${var.database_name}"
   database_user                    = "${var.database_user}"
   database_password                = "${var.database_password}"
+  db_subnet_group_name             = "${module.survey-runner-vpc.database_subnet_group_name}"
+  database_identifier              = "${var.env}-digitaleqrds"
+  rds_security_group_name          = "${var.env}-rds-access"
+}
+
+module "author-database" {
+  source                           = "./survey-runner-database"
+  env                              = "${var.env}"
+  aws_access_key                   = "${var.aws_access_key}"
+  aws_secret_key                   = "${var.aws_secret_key}"
+  vpc_id                           = "${module.survey-runner-vpc.vpc_id}"
+  application_cidrs                = "${concat(var.ecs_application_cidrs, var.application_cidrs)}"
```

Comment:
This only needs access from ECS so we can change this to `"${var.ecs_application_cidrs}"`